projects/1.19/assets/farmers-respite/farmersrespite/lang/zh_cn.json

```
@@ -0,0 +1,104 @@
+{
+	"block.farmersrespite.black_candle_coffee_cake": "插上黑色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.black_tea_leaves_sack": "袋装红茶叶",
+	"block.farmersrespite.blue_candle_coffee_cake": "插上蓝色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.brown_candle_coffee_cake": "插上棕色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.candle_coffee_cake": "插上蜡烛的咖啡蛋糕",
+	"block.farmersrespite.coffee_beans_sack": "袋装咖啡豆",
+	"block.farmersrespite.coffee_bush": "咖啡丛",
+	"block.farmersrespite.coffee_bush_top": "咖啡丛顶部",
+	"block.farmersrespite.coffee_cake": "咖啡蛋糕",
+	"block.farmersrespite.coffee_stem": "咖啡茎",
+	"block.farmersrespite.coffee_stem_double": "咖啡茎底部",
+	"block.farmersrespite.coffee_stem_middle": "咖啡茎中部",
+	"block.farmersrespite.cyan_candle_coffee_cake": "插上青色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.gray_candle_coffee_cake": "插上灰色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.green_candle_coffee_cake": "插上绿色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.green_tea_leaves_sack": "袋装绿茶叶",
+	"block.farmersrespite.kettle": "水壶",
+	"block.farmersrespite.light_blue_candle_coffee_cake": "插上淡蓝色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.light_gray_candle_coffee_cake": "插上淡灰色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.lime_candle_coffee_cake": "插上黄绿色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.magenta_candle_coffee_cake": "插上品红色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.orange_candle_coffee_cake": "插上橙色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.pink_candle_coffee_cake": "插上粉红色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.potted_coffee_bush": "咖啡丛盆栽",
+	"block.farmersrespite.potted_tea_bush": "茶树盆栽",
+	"block.farmersrespite.potted_wild_coffee_bush": "野生咖啡丛盆栽",
+	"block.farmersrespite.potted_wild_tea_bush": "野生茶树盆栽",
+	"block.farmersrespite.purple_candle_coffee_cake": "插上紫色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.red_candle_coffee_cake": "插上红色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.rose_hip_pie": "玫瑰果派",
+	"block.farmersrespite.small_tea_bush": "茶籽",
+	"block.farmersrespite.tea_bush": "茶树",
+	"block.farmersrespite.white_candle_coffee_cake": "插上白色蜡烛的咖啡蛋糕",
+	"block.farmersrespite.wild_coffee_bush": "野生咖啡丛",
+	"block.farmersrespite.wild_tea_bush": "野生茶树",
+	"block.farmersrespite.wither_roots": "凋零根茎",
```

```suggestion
	"block.farmersrespite.wither_roots": "凋灵根茎",
```
1193的官译wither rose从`凋零玫瑰`变为了`凋灵玫瑰`